Userland/Applications/GamesSettings/CardSettingsWidget.cpp

```diff
@@ -84,6 +89,9 @@ bool CardSettingsWidget::set_card_back_image_path(String const& path)
 
 String CardSettingsWidget::card_back_image_path() const
 {
-    auto card_back_image_index = m_card_back_image_view->selection().first();
+    auto& card_back_selection = m_card_back_image_view->selection();
+    GUI::ModelIndex card_back_image_index = m_last_selected_card_back;
+    if (!card_back_selection.is_empty())
+        card_back_image_index = card_back_selection.first();
```

Comment:
`Optional` has a handy `value_or()` method that you can use for situations like this.

```suggestion
    GUI::ModelIndex card_back_image_index = card_back_selection.value_or(m_last_selected_card_back);
```

Reply:
It appears the type of symbol `card_back_selection` does not implement or extend `Optional`: `CardSettingsWidget` calls `selection` from a `GUI::IconView`, which inherits `GUI::AbstractView`, which contains the method `selection` returning a `GUI::ModelSelection`. `GUI::ModelSelection` appears neither to implement `value_or` nor descend from any interface or class implementing `value_or`.

Reply:
Ah whoops, you're right! Sorry about that.